service/src/main/java/bio/terra/pipelines/app/controller/PipelinesApiController.java

```diff
@@ -58,4 +97,82 @@ static ApiPipeline pipelineToApi(Pipeline pipelineInfo) {
         .displayName(pipelineInfo.getDisplayName())
         .description(pipelineInfo.getDescription());
   }
+
+  // Pipelines jobs
```

Comment:
probably worth expanding on this comment since its probably the most integral endpoint we have

Reply:
added descriptions to each controller method below - is this what you had in mind, or is there still missing context?